File: angular-chatty/src/app/model/message.ts
export interface Message {
    id: number;
    senderId: number;
    senderProfile: any;
    sentToId: number;
    chatId: number;
    messageDate: Date;
    text: string;
    mediaId: number;
    media: any;
    created: Date;
    updated: Date;
    disabled: boolean;
    deleted: boolean;
}

File: src/main/java/com/example/chatty/models/dto/Message.java
package com.example.chatty.models.dto;

import lombok.Data;

import java.sql.Timestamp;
import java.time.LocalDateTime;

@Data
public class Message {

    private Long id;
    private Long senderId;
    private Profile senderProfile;
    private Long sentToId;
    private Long chatId;
    private LocalDateTime messageDate;
    private String text;
    private Long mediaId;
    private Media media;
    private LocalDateTime created;
    private LocalDateTime updated;
    private boolean disabled;
    private boolean deleted;

}
